model_info/ptwiki.wp10.md

```diff
@@ -15,71 +15,71 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-11-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=8921):
+	counts (n=8574):
```

Comment:
This has dropped a lot.  I wonder if it is related to some pages getting deleted?

Reply:
I confirmed that https://github.com/wikimedia/articlequality/blob/master/articlequality/utilities/fetch_text.py#L110 does follow redirects so pagemoves shouldn't case a big loss in observations.